### DISCUSSION TITLE
Add ability to remove or revoke uninitialized devices

### DISCUSSION THREAD

**Discussion by v0tti** [pretix/pretix#2143] [Ideas]
At the moment only initialized devices may be revoked from an organizer. If a device is not initialized it can not be revoked or deleted. This could potentially be a problem if the token is already given out and could gain access to personal data (check-in lists).
The only workaround at the moment is to either assign the device to a dummy event or manually initialize the token on a different device and then revoke its access.

**Comment by raphaelm** [comment]
good catch!
